Assets/Scripts/Ball/BallMover.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(GroundCheckParams))]
+public class BallMover: MonoBehaviour
+{
+    [SerializeField] private float _jumpForce;
+    [SerializeField] private float _movingSpeed;
+    [SerializeField] private Vector2 _startPosition;
+    private GroundCheckParams _groundCheckParams;
+
+    private bool _isGrounded;
+    private Rigidbody2D _body;
+    private float _movement;
+    private const string Jump = "Jump"; 
```

Comment:
https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods